modules/openapi-generator/src/main/resources/typescript-rxjs/runtime.mustache

```diff
@@ -35,19 +35,13 @@ export class Configuration {
     }
 
     get apiKey(): ((name: string) => string) | undefined {
-        const apiKey = this.configuration.apiKey;
-        if (!apiKey) {
-            return undefined;
-        }
-        return typeof apiKey === 'string' ? () => apiKey : apiKey;
+        const { apiKey } = this.configuration;
+        return apiKey ? (typeof apiKey === 'string' ? () => apiKey : apiKey) : undefined;
```

Comment:
I don't think double ternary is appropriate there. It's obfuscating the conditional, it makes it less readable. I've addressed the nully coalescing in #5329, and I think webpack can take care of the compressing the code.